### ISSUE TITLE
EMu will not consistently create new lot numbers

### ISSUE THREAD

**Issue by ljwalker** [LACMIP/emu#39]
**What module(s) and tab(s) does this bug affect?**
_Catalogue: Invert Paleo_

**Describe the problem**
During cataloging, EMu will not consistently create new numbers in the _Lot No._ field. This problem occurs immediately after a number is specified in the _Locality_ field, after which "tab" immediately is keystroked. The only apparent solution is to close and reopen the _Catalogue_ module, which is very inefficient.

**Describe the solution you'd like**
EMu should be able to create new lot numbers. This step is fundamental to the cataloging process.

**Comment by wlmertz** [comment]
Axiell has added a Local Field for SitSiteNumber to mimic the setup for Marine Invertebrates.  Lets see if this resolves the issue with the Lot Number not incrementing.  Please follow-up.

**Comment by wlmertz** [comment]
10/02/2019 updated client on Marconi.  Please contact me if numbering issue persists.

**Comment by ljwalker** [comment]
Daniel and Shawn report that they are still regularly experiencing this issue in EMu.

**Comment by wlmertz** [comment]
Thanks.  I'll contact Axiell about this again.  It is difficult to diagnose since it cannot be reproduced.  Next time this occurs, please contact me immediately so I can test that specific record.

**Comment by dmarkbreiter** [comment]
@wlmertz 
Below is an example of the bug, as I've encountered it. When the record's locality number is changed to '40374' and a lot number of '46' is assigned by EMu, "This Bar Code Already Exists" error message is displayed and the record can't be saved. 

The record was edited using Heather Chaides's account. 

![LotNumberBugPic](https://user-images.githubusercontent.com/58999191/74373640-b629a280-4d91-11ea-9abe-9208757ec029.PNG)

**Comment by wlmertz** [comment]
This is because we've disabled editing for Lot No.   EMu responded as expected.  40374 Lot No. 46 does indeed already exists, and therefore EMu will not add it until the Lot No. is updated (which it can't be because Lot No. editing is currently disabled.)  I'll modify the permissions to allow editing.  Should editing be allowed for all IP Cataloging group and Invertebrate Paleontology group?

**Comment by wlmertz** [comment]
I've changed the permissions.  Please test again.  Also, if editing the Locality field in Catalog is to be allowed, then Lot No. cannot be read-only.

**Comment by ljwalker** [comment]
> Should editing be allowed for all IP Cataloging group and Invertebrate Paleontology group?

No, please do not allow IP users (other than Invertebrate Paleontology admin) to be able to edit lot numbers.

**Comment by ljwalker** [comment]
> Should editing be allowed for all IP Cataloging group and Invertebrate Paleontology group?

No, no one in IP should be able to edit lot numbers at this time. 

Daniel's immediate issue has been resolved.

**Comment by wlmertz** [comment]
Thanks.  Only group Invertebrate Paleontology can edit Lot No. , and therefore only group Invertebrate Paleontology should change the Locality of a Catalog record.  I will disable editing of Locality in Catalog for all other groups as well.
